src/Login/LoginForm.tsx

```diff
@@ -55,18 +55,23 @@ export const LoginForm: React.FC<LoginFormProps> = () => {
 
             if (resp.status === 200) {
                 setLoginAlertMsg({ message: resp.data?.msg || 'Logged in!', variant: 'success' });
-                // TODO: Create a User class to massage and error-handle these fields.
-                Cookie.set(CookieEnum.USERTYPE, getUserRoleFromServer(resp.data.data.roleId));
-                Cookie.set(CookieEnum.USERID, resp.data.data.userId);
-                Cookie.set(CookieEnum.USERNAME, `${resp.data.data.firstName} ${resp.data.data.lastName}`);
+                session.userId = resp.data.data.userId;
+                session.userType = getUserRoleFromServer(resp.data.data.roleId);
+                session.userUUID = resp.data.data.uuid;
+                session.username = `${resp.data.data.firstName} ${resp.data.data.lastName}`;
+                // TODO delete this code, moving to local preferences but want to keep this for debugging
```

Comment:
You can delete this now.

Reply:
Saving it since this is a big change, I'll clean it up in dev after it is tested in stage